### PR TITLE
Warn if udunits2 fails

### PR DESCRIPTION
**Pull request by ocefpaf**
Solves https://github.com/ioos/compliance-checker/issues/265#issuecomment-215791057

Ping @lukecampbell

### CODE REVIEW

File: cf_units/__init__.py [hunk 1]
```diff
@@ -220,10 +220,15 @@
 # load the libudunits2 shared library
 #
 if _lib_ud is None:
-    _lib_ud = config.get_option(
+    udunits2_c = config.get_option(
```
Comment (rhattersley) [inline cf_units/__init__.py]: The convention so far has been for these kinds of helper variables to have an underscore prefix to mark them as private and stop them from turning up in tab-completion.

Comment (ocefpaf) [inline cf_units/__init__.py]: Thanks @rhattersley. I just fixed that.

**Comment by rhattersley** [comment]
Thanks @ajdawson :+1:

**Comment by ocefpaf** [comment]
Thanks for merging @rhattersley. Can you take a look at #54 too. It is also minor changes.

**Pull request merged**